src/Type/Union/ContentRevisionUnion.php

```diff
@@ -36,6 +36,7 @@ function( $post_type ) {
 				'ContentRevisionUnion',
 				[
 					'typeNames'   => $type_names,
+					'description' => 'Union between the Post and Page types',
```

Comment:
translation

Also, this can be more than `Post` and `Page` Types. 

Maybe something like: 

`A union of Content Node Types that support revisions` or something like that?